src/node_contextify.cc

```diff
@@ -728,6 +736,26 @@ class ContextifyScript : public BaseObject {
   }
 
 
+  static void CreateCodeCache(const FunctionCallbackInfo<Value>& args) {
+    Environment* env = Environment::GetCurrent(args);
+    ContextifyScript* wrapped_script;
+    ASSIGN_OR_RETURN_UNWRAP(&wrapped_script, args.Holder());
+    Local<UnboundScript> unbound_script =
+        PersistentToLocal(env->isolate(), wrapped_script->script_);
+    const ScriptCompiler::CachedData* cached_data =
+      ScriptCompiler::CreateCodeCache(unbound_script);
+    if (cached_data == nullptr) {
+      THROW_ERR_CREATE_CODE_CACHE_FAILED(env, "Creating code cache failed.");
+    } else {
+      MaybeLocal<Object> buf = Buffer::Copy(
+          env,
+          reinterpret_cast<const char*>(cached_data->data),
+          cached_data->length);
+      args.GetReturnValue().Set(buf.ToLocalChecked());
```

Comment:
I think you need to delete `cache_data`. Or even better, use unique_ptr around it.